client/my-sites/stats/stats-module/style.scss

```diff
@@ -449,13 +449,13 @@ ul.module-header-actions {
 	@include breakpoint-deprecated( ">480px" ) {
 		.stats-module & td.has-no-data:hover, // 1
 		tbody tr:hover td,
-		tbody tr:hover th {
-			background: var(--color-neutral-0);
+		.stats-module & tbody tr:hover th {
```

Comment:
```suggestion
		.stats-module & tbody tr:hover th:first-child {
```

Applying a more specific selector here should force `th` elements to use the desired background, like so:

<img width="1071" alt="image" src="https://user-images.githubusercontent.com/4044428/210886514-a0440f7a-da25-4a24-9b3e-faf19144807c.png">